step-api/step-api-keyword/src/main/java/step/handlers/javahandler/Keyword.java

```diff
@@ -63,4 +65,14 @@
 	 * @return the reference to file with implementing plan (for composite keyword only)
 	 */
 	String planReference() default "";
+
+	/**
+	 *
+	 * @return the routing criteria which can be either <ul>
+	 *     <li>empty: use default routing on Step</li>
+	 *     <li>{@link #EXECUTE_ON_CONTROLLER}: execute the keyword directly on the controller rather than on agents</li>
```

Comment:
When renaming, remember to adjust the link as well (if not done automatically by refactoring)